utils/utils.go

```diff
@@ -26,6 +26,8 @@ func RenderMarkdown(text string) (string, error) {
 	if isColorEnabled() {
 		style = "dark"
 	}
+	// This ensures that carriage returns before the newlines are omitted
```

Comment:
This comment currently just describes in words exactly what the code does. Generally, such comments are not very useful. It's a much better idea to use comments to communicate the _intent_ behind some code.

Since the intent is to ensure that no carriage returns are present in the result, maybe the comment could reflect that? e.g.

```
// Glamour rendering preserves carriage return characters in code blocks, but
// we need to ensure that no such characters are present in the output
```

Reply:
True. I wanted to relay what the code does, instead that should be obvious. I'll reflect that